portal/src/components/ui/Canvas.tsx

```diff
@@ -6,54 +6,18 @@ import laboratory from '@/assets/laboratory.png';
 import { Zone } from '@/types/canvas';
 import useScreenStore from '@/stores/screenStore';
 import { Screen } from '@/types/gameConsole';
+import { Zones } from './zones';
 
 const mouseClick = {
   x: 0,
   y: 0,
 };
+
 const inBox = (x: number, y: number, box: Zone): boolean => {
   return x >= box.x && x <= box.x + box.w && y >= box.y && y <= box.y + box.h;
 };
 
-const zoneArray: Zone[] = [
```

Comment:
Moved these to a file and made them into a map so we can get zones by screen.

Reply:
love it